### PR TITLE
Move artifacts upload summary out of uploader implementation

### PR DESCRIPTION
**Pull request by nfalco79**
We had move the execution of our jobs from master to slave but happens unexpected exception in URLMaker when try to build URL.
The exception was hidden because, the only exception permitted on a FileCallable are IOException (or InterrupException). Looking the console the issue happens in makeArtifacts method of UploaderV3. In theory all server data in the callable should be available and should never happens any URLException anyway I had move the artifacts upload summary out of Uploader implementations since it's a logger decoration which has nothing to do with the uploaders logic.

So in short changes are:
- update to latest 2.x plugins pom
- move upload artifacts summary creation out of Uploader
- standardize the summary regardless of the version of the Uploader implementation

### CODE REVIEW

**Comment by nfalco79** [comment]
After merge to master could you also perform a new release please?

**Comment by ctron** [comment]
I will have a look at this early next week.

**Comment by ctron** [comment]
Thanks for the contribution. I did release 0.6.0 a few minutes ago.
